assignment-04.Rmd

```diff
@@ -1,67 +1,155 @@
 ---
-title: 'Assignment 4: exploratory data analysis and linear models (8 marks)'
+title: 'Assignment 4: Exploration, linear and mixed-effects models'
 output:
     html_document:
         toc: false
 ---
 
-```{r setup, echo=FALSE}
-knitr::opts_chunk$set(eval = FALSE)
-```
-
 *To submit this assignment, upload the full document on blackboard,
 including the original questions, your code, and the output. Submit
 you assignment as a knitted `.pdf` (prefered) or `.html` file.*
 
-1. Visualization (2 marks)
+1. Visualization (3 marks)
 
-    Import the tidyverse library. We will be using the same beaver1 dataset that we used in last week's assignment. 
+    Import the tidyverse library. We will be using the same beaver1 dataset that
+    we used in last week's assignment.
 
     ```{r message=FALSE, warning=FALSE}
     library(tidyverse)
     ```
 
-    a. Using what you learned during your last assignment, turn the "activ" column into a categorical variable (factor). Then, create a plot with body temperature on the x-axis. Visualize body temperature separately by whether the beaver is active on the same graph. Hint: we learned about a new useful function that does this during our exploratory data lesson. (1 mark)
-
-    b. Create a dataframe (summary table) that counts the number of temperature recordings that fall into bins with a width of 0.1. Group by day. (1 mark)
-
-2. Outliers and Missing Values (2.5 marks)
+    a. Create a histogram to visualize the distribution of the beavers' body
+    temperatures, separating the temperature data based on the beaver's activity level.
+    (after transforming it into a categorical variable the way you did for your
+    last assignment). Describe the properties of the distribution. When
+    creating this plot for the purpose of evaluating temp temperature, what
+    argument did you adjust and why? (1 mark)
 
-    a. Identify any temperature scores that are two standard deviations away from the mean. Replace them with NA's (missing values). (1 mark)
+    b. What type of variables are temperature and time of day? With this in
+    mind, create a visualization that will help you get a better understanding
+    of the relationship between these variables. (0.5 mark)
 
-    b. Run the following code:
+    c. Create a single box plot that to simultaneously visualise temperature,
```

Comment:
s/plot that to/plot to/g

or equivalent

Reply:
Your syntax is :fire: :joy: